tests/typescript_import_export/type-modifier.ts

```diff
@@ -0,0 +1,5 @@
+import type { SomeThing } from "./some-module.js";
+export type { SomeThing };
+
+import type Foo from "./foo.js";
+export type Foo;
```

Comment:
is this a valid syntax? (ref: https://www.typescriptlang.org/play/?ssl=1&ssc=16&pln=1&pc=21#code/KYDwDg9gTgLgBDAnmYcBiEIG4BQQ)

Reply:
Oops, no.